### PR TITLE
feat(media): call ui updates

### PR DESCRIPTION
**Pull request by jasonwoodland**
**What this PR does** 📖
- Improve volume slider
- Improve drag bar
- Calculate optimal MediaUser dimensions
- Refactor calling UI
- Update talking indicator

**Which issue(s) this PR fixes** 🔨
- #4526

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: components/views/media/user/User.less [hunk 1]
```diff
@@ -75,9 +90,9 @@
     height: @indicator-size;
     opacity: 0.5;
     border-radius: 100%;
-    background-color: @foreground-alt;
-    box-shadow: -2px 2px 30px rgba(18, 18, 22, 0.45),
-      0px 1px 2px rgba(43, 43, 59, 0.65);
+    // background-color: @foreground-alt;
+    // box-shadow: -2px 2px 30px rgba(18, 18, 22, 0.45),
+    //   0px 1px 2px rgba(43, 43, 59, 0.65);
```
Comment (stavares843) [inline components/views/media/user/User.less:95]: do we need this commented out?

Comment (WanderingHogan) [reply components/views/media/user/User.less:95]: commented out doesn't matter right now

**Comment by WanderingHogan** [comment]
the only issue I saw was there was three video windows on a direct call

**Comment by jasonwoodland** [comment]
> the only issue I saw was there was three video windows on a direct call

@WanderingHogan got a screenshot? probably the corrupt video state problem I mentioned before, the video/screen stream muted state is not being initialised correctly.

**Comment by stavares843** [comment]
> the only issue I saw was there was three video windows on a direct call

this is also on dev, phil noticed this as well yesterday


https://user-images.githubusercontent.com/29093946/188034891-b9eb2b0d-5b37-4b9e-8c45-1a95de16cbfc.mov

**Comment by WanderingHogan** [comment]
the rebase on this one isn't so bad, conflicts are pretty minor, but it looks like a lot of other things changed a little bit so we will need to test it pretty well

File: components/views/media/Media.vue [hunk 2]
```diff
@@ -52,14 +65,98 @@ export default Vue.extend({
         this.$store.commit('ui/setCallHeight', value)
       },
     },
+    call(): Call | undefined {
```
Comment (WanderingHogan) [inline components/views/media/Media.vue:68]: something in here

**Comment by jasonwoodland** [comment]
> > the only issue I saw was there was three video windows on a direct call
> 
> this is also on dev, phil noticed this as well yesterday
> 
>  Screen_Recording_2022-08-31_at_11.16.09_AM.1.mov 
> the flow was when using screen share would do a extra window as phil mentioned on dev

This was due to invalid muted stream state. Will be fixed by another PR

**Review by josephmcg — CHANGES_REQUESTED** [review]
the volume sliders on the settings page use the same component as the call (for some reason lol). may need some refactoring
image

also get this error on receiver side
image

**Comment by josephmcg** [comment]
still getting the same error from before. Looks like the refs inside of the mounted logic are coming up undefined (for one side only)

**Review by josephmcg — APPROVED** [review]

**Pull request merged**